ui/components/app/advanced-gas-controls/advanced-gas-controls-row.component.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import NumericInput from '../../ui/numeric-input/numeric-input.component';
+import Tooltip from '../../ui/tooltip/tooltip';
+
+export default function AdvancedGasControlsRow({
+  titleText,
+  tooltipText,
+  titleDetailText,
+  error,
+}) {
+  return (
+    <div
+      className={classNames('advanced-gas-controls__row', {
+        'advanced-gas-controls__row--error': error,
+      })}
+    >
+      <label>
+        <div className="advanced-gas-controls__row-heading">
+          <div className="advanced-gas-controls__row-heading-title">
+            <h6>{titleText}</h6>
+            {tooltipText && (
+              <Tooltip
+                wrapperClassName="advanced-gas-controls__row-heading-tooltip"
+                title={tooltipText}
+                position="top"
+                arrow
+              >
+                <i className="fa fa-info-circle" />
+              </Tooltip>
+            )}
+          </div>
+          {titleDetailText && (
+            <div className="advanced-gas-controls__row-heading-detail">
```

Comment:
[req] Lets use the <Typography> component for these new text-only classes, and use HTML tags more appropriate for text content.